caf-audit-binding-elasticsearch/src/main/java/com/hpe/caf/auditing/elastic/ElasticAuditIndexManager.java

```diff
@@ -148,13 +154,19 @@ private void createIndex(String indexName){
                     throw new IllegalStateException(errorMessage);
                 }
             }
-        } catch (ResourceAlreadyExistsException raee) {
+        } catch (final IOException io) {
+            //TODO ANDY - NEED to verify the exception was caused by the index already existing.
             //  Index already exists. Just ignore.
-            LOG.debug(indexAlreadyExistsMessage, raee);
+            LOG.debug(indexAlreadyExistsMessage, io);
         }
     }
 
     private boolean indexExists(final String indexName) {
-        return transportClient.admin().indices().prepareExists(indexName).execute().actionGet().isExists();
+        try {
+            return restHighLevelClient.indices().exists(new GetIndexRequest(indexName), RequestOptions.DEFAULT);
+        } catch (final IOException e) {
+            LOG.error(e);
+            return false;
```

Comment:
This seems dodgy.  I think we should just remove all this index checking stuff and use index templates.

Reply:
In another Jira I mean.

Reply:
I've raised [SCMOD-7034](https://portal.digitalsafe.net/browse/SCMOD-7034).  We can consider that option during that Jira.